br/pkg/task/stream.go

```diff
@@ -1103,6 +1104,10 @@ func restoreStream(
 		return errors.Annotate(err, "failed to clean up")
 	}
 
+	if err = client.InsertGCRows(ctx); err != nil {
```

Comment:
If we had error here. the cluster still can be used. so I think we can put it at last after `client.SaveSchemas`. WDYT?